app/assets/stylesheets/application.scss

```diff
@@ -19,16 +19,20 @@ html, body, .landing {
 
 .container--banner {
     height: 80px;
+    margin-left: 50px;
 }
 
 .title {
-    margin-left: 50px;
     margin-top: 20px;
     font-family: "Roboto", sans-serif;
     color: #77777;
     font-size: 3em;
 }
 
+.success {
+    color: #009688;
```

Comment:
Color literals like `#009688` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces